functorch/_src/eager_transforms.py

```diff
@@ -670,7 +670,9 @@ def safe_unpack_dual(dual, strict):
     return primal, tangent
 
 
-def jvp(func, primals, tangents, *, strict=False):
+def jvp(
+    func: Callable, primals: Any, tangents: Any, *, strict: bool = False, has_aux: bool = False
+) -> Union[Tuple[Any, Any], Tuple[Any, Any, Any]]:
```

Comment:
I'm not sure that having the type of the return is helpful to users, what do you think?

Reply:
I can remove it as we do not run any mypy checks. I added it as added the types for input signature.